### ISSUE TITLE
doesn't write xml

### ISSUE THREAD

**Issue by zonky2** [RhymeDigital/isotope_feeds#2]
Hi,

I install the curret package at Contao 3.4.5 and Isotope 2.2.4-3 and set all configuration parts for feeds...

At the system log I found

```
Details:     Generated product feed share/my-products-googlebase.xml
Funktion:     Rhyme\IsotopeFeeds::generateFeeds  
```

but I doesn't found the xml at /share...

One more question: the "Feed Reader Page" should the page with details (with module reader) or the page with listing (module listing)?

Thanks!

**Comment by zonky2** [comment]
at config site of product-type I see this

![zwischenablage17](https://cloud.githubusercontent.com/assets/1045318/8267292/e3118d48-175a-11e5-86eb-68fc2cec455e.jpg)

and during the save of a product came the follow error

```
Fatal error: Call to undefined method Rhyme\Backend\IsotopeProduct\FeedCallbacks::cacheProduct() in /htdocs/contao/system/modules/isotope/drivers/DC_ProductData.php on line 769  
```

**Comment by zonky2** [comment]
the button "Refresh feed cache" and the popup with the start button, brings the follow login mask

![zwischenablage18](https://cloud.githubusercontent.com/assets/1045318/8277986/e99733a2-18c7-11e5-9967-406b49a52233.jpg)

**Comment by bytehead** [comment]
@xantippe You won't get any feeds, if there is no data provided in the products.
The feed showed up, after I filled in the metadata for the feed.

![bildschirmfoto 2016-02-29 um 16 15 58](https://cloud.githubusercontent.com/assets/754921/13398717/d4f318b8-deff-11e5-859b-92d7dcba7940.png)

**Comment by bytehead** [comment]
This gets all fixed with #4.
